lib/Doctrine/DBAL/Query/From.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Query;
+
+/**
+ * @internal
+ */
+final class From
+{
+    /** @var string|null */
```

Comment:
What's the use case when the table is null?

Reply:
I have no idea. I really just converted the existing code for now.